<launch>
  <!-- Load joint controller configurations from YAML file to parameter server -->
  <rosparam file="$(find robocup_control)/config/player_control.yaml" command="load" />
  <rosparam file="$(find robocup_control)/config/player_hw.yaml" command="load" />
  <!-- Load joint limit configurations from YAML file to parameter server -->
  <!--<rosparam file="$(find robocup_control)/config/player_joint_limits.yaml" command="load">-->
  <!-- Start Player Thread-->
  <node name="player_thread" pkg="robocup_control" type="player_thread" respawn="false" output="screen" ns="/robocup_control"/>
 
  <!-- Load the controllers -->
  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" ns="/robocup_control" args="joint_state_controller drive_controller"/>
  <!-- Convert joint states to TF transforms -->
  <!--<node name="player_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" respawn="false" output="screen">
    <remap from="/joint_states" to="/player/joint_states" />
  </node>-->
</launch>
<!-- add parameter switch to turning the dummy service on or the real service on -->
